typer/rich_utils.py

```diff
@@ -487,7 +488,17 @@ def _print_commands_panel(
     )
     # Define formatting in first column, as commands don't match highlighter
     # regex
-    commands_table.add_column(style="bold cyan", no_wrap=True)
+    commands_table.add_column(
+        style="bold cyan",
+        no_wrap=True,
+        width=cmd_len,
+    )
+
+    # A big ratio makes the description column be greedy and take all the space
+    # available instead of allowing the command column to grow and misalign with
+    # other panels.
+    commands_table.add_column("Description", justify="left", no_wrap=False, ratio=10)
+    commands_table.add_column("xx", style="bold yellow", justify="left", no_wrap=False)
```

Comment:
Why is this `"xx"` column needed? On my end, things seem to still work correctly without it.

Reply:
Agreed, it seems this last line is not necessary, I just removed it.